src/store/datasets-images.ts

```diff
@@ -86,14 +84,6 @@ export const useImageStore = defineStore('images', {
         del(this.dataIndex, id);
         del(this.metadata, id);
         removeFromArray(this.idList, id);
-
-        // Remove the data views
-        const viewStore = useViewStore();
-        const viewConfigStore = useViewConfigStore();
-
-        viewStore.viewIDs.forEach((viewID: string) => {
-          viewConfigStore.removeData(id, viewID);
-        });
```

Comment:
=)

Reply:
Finally (re)moved! Had to avoid cyclic dependencies.